config/main.yml

```
@@ -79,6 +79,11 @@ filterFeeds:
     filterFeedEmoji: '🎉'
     title: '{{num}} tickets have just been resolved as Fixed!'
     titleSingle: This ticket has just been resolved as Fixed!
+  - jql: project = MC AND status changed AFTER -1m AND status = "In Progress"
+    channel: '666349583227682819'
```

```suggestion
    channel: '666349583227682819'
    interval: 30000
```
Not sure if this was left out on purpose :eyes: